czkawka_gui/src/gui_structs/gui_about.rs

```diff
@@ -51,7 +51,7 @@ impl GuiAbout {
             "Mek101",
             "Michael Grigoryan",
             "Nikita Karamov",
-            "Proprietary Chrome-chan",
+            "TheEvilSkeleton",
```

Comment:
Can you add in comment `// Proprietary Chrome-chan`? 
This will help me to not forget to replace the name with the proper one in future changes